data/data_download/wdpa/Makefile

```diff
@@ -0,0 +1,71 @@
+.PHONY: import
+MAKEFLAGS := --jobs=$(shell nproc)
+MAKEFLAGS += --output-sync=target
+
+URL := $(shell curl -sS 'https://www.protectedplanet.net/downloads' \
+		-H 'content-type: application/json;charset=UTF-8' \
+		--data-raw '{"domain":"general","format":"shp","token":"wdpa","id":51216}' \
+		--compressed | jq '.url')
+# @debt
+# USER_TOKEN := $(shell curl -sS 'https://mgis.vizzuality.com/auth/login' \
+# 		-H 'content-type: application/json;charset=UTF-8' \
+# 		--data-raw '{"email":"aa@example.com","password":"aauserpassword"}' \
+# 		--compressed | jq '.token')
+# need to add as the main admin user through auth and calling for it.
```

Comment:
note for the future - I would avoid requesting a token here... In general, I think we should either request a normal token *from outside the container* before running it and then pass this in at runtime, or generate a long-lived token and configured that as a secret.
I would not even pass in to the container a valid API username and password at runtime - though this is a bit of a personal preference thing, to a certain extent.